data-prepper-plugins/dynamodb-source/src/main/java/org/opensearch/dataprepper/plugins/source/dynamodb/DynamoDBSource.java

```diff
@@ -58,10 +59,16 @@ public DynamoDBSource(final PluginMetrics pluginMetrics,
         this.sourceConfig = sourceConfig;
         this.pluginFactory = pluginFactory;
         this.acknowledgementSetManager = acknowledgementSetManager;
+        this.acknowledgementsEnabled = sourceConfig.isAcknowledgmentsEnabled();
 
         clientFactory = new ClientFactory(awsCredentialsSupplier, sourceConfig.getAwsAuthenticationConfig(), sourceConfig.getTableConfigs().get(0).getExportConfig());
     }
 
+    @Override
+    public boolean areAcknowledgementsEnabled() {
```

Comment:
You need to pull from main and rebase your branch against main to get rid of this old commit. If it still shows up then create a new branch off main and cherry-pick the commit for the worker change to it